docs/site/HasMany-relation.md

```diff
@@ -189,4 +189,18 @@ export class CustomerOrdersController {
 }
 ```
 
-<!--- TODO: Add details about `orderData` variable typing issue --->
+In LoopBack 3, methods for accessing related models were part of the source
+model class i.e. `customer.orders.find(...)`. We recommend to create new
```

Comment:
In LB 3.x, the REST API for relations was exposed via static methods with the name following the pattern `__{methodName}__{relationName}__`, for example `Customer.__find__orders`.

Reply:
Oh I was looking at https://loopback.io/doc/en/lb3/HasMany-relations.html#methods-added-to-the-model when I wrote that. But you're right, this is about REST APIs 👍